README.md

```diff
@@ -73,3 +73,21 @@ EOF
 ```
 kubectl create -f <rdma-app.yaml>
 ```
+
+# RDMA Shared Device Plugin with NFD (Node Feature Discovery)
```

Comment:
Repharase:

# RDMA shared device plugin deployment with node labels

RDMA shared device plugin should be deployed on nodes that:
1. Have RDMA capable hardware
2. RDMA kernel stack is loaded

To allow proper node selection [Node Feature Discovery (NFD)](https://github.com/kubernetes-sigs/node-feature-discovery)  can be used to discover the node capabilities, and expose them as node labels.